gravityflow/gravityflow
# Refine date/time filter for entry detail info box and {current_step} merge tag

See [HS#6545](https://secure.helpscout.net/conversation/637376929/6545/?folderId=1776095) for further details.
Extend locations where filter(s) apply for date display
**Issues:** 
1) In the workflow workflow entry detail info box the filter gravityflow_date_format_entry_detail was not applied to expiration date in same fashion as the creation, last update values.
2) In the {current_step} merge tag there are several date/time modifiers which did not include a filter to modify the format. 

**Resolution:** 
1) Apply the filter before the expiration date is output.
2) Add gravityflow_date_format_current_step_merge_tag filter and apply to modifier output for expiration, schedule (all 3 schedule types), and start.

**Testing Instructions:**
* Create a workflow with a step that has expiration
* Note that prior to PR the expires date does not return the modified format.
* Add the following code to a test functions.php or plugin file and note that all do adjust.
add_filter( 'gravityflow_date_format_entry_detail', 'sh_current_step_date_format', 10, 1 );
add_filter( 'gravityflow_date_format_current_step_merge_tag', 'sh_current_step_date_format', 10, 1 );
```
function sh_current_step_date_format( $date_format ){
	$date_format = 'M d, Y';
	return $date_format;
}
```
* Rejoice.
